docs/topics/tour/kotlin-tour-classes-part-1.md

```diff
@@ -0,0 +1,279 @@
+    contact.printId()           //Calls member function printId()
+    //1
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-member-function-kotlin"}
+
+## Data classes
+
+Kotlin has **data classes** which are particularly useful for storing data. Data classes have the same functionality as 
+classes, but they come automatically with additional member functions. These member functions allow you to easily copy 
+instances of a class, compare instances, print the instance to readable output, and more. As these functions are
+automatically available, you don't have to waste time writing the same boilerplate code for each of your classes.
+
+To declare a data class, use the keyword `data`:
+```kotlin
+data class User(val name: String, val id: Int)
+```
+
+All the predefined member functions of a data class are listed in the table below:
+
+| **Function**       | **Description**                                                                                                |
+|--------------------|----------------------------------------------------------------------------------------------------------------|
+| `copy()`           | Creates a class instance by copying another, potentially with some different properties.                       |
+| `componentN()`     | Accesses properties of the class in their order of declaration. E.g. `component1` accesses the first property. |
+| `equals()` or `==` | Compares instances of a class.                                                                                 |
+| `hashCode()`       | Exposes the hash code of an instance, for comparison.                                                          |
+| `toString()`       | Prints a readable string of the class instance and its properties.                                             |
+
+See the following sections for examples of how to use each function:
+* [Copy instance](#copy-instance)
+* [Access properties](#access-data-class-properties)
+* [Compare instances](#compare-instances)
+* [Expose hash code](#expose-hash-code)
+* [Print as string](#print-as-string)
+
+### Copy instance
+
+```kotlin
+data class User(val name: String, val id: Int)
+
+fun main() {
+    //sampleStart
+    val user = User("Alex", 1)
+    val secondUser = User("Alex", 1)
+    val thirdUser = User("Max", 2)
+    
+    println(user.copy())       //Creates an exact copy of user
+    //User(name=Alex, id=1)
+    println(user.copy("Max"))  //Creates a copy of user with name: "Max"
+    //User(name=Max, id=1)
+    println(user.copy(id = 3)) //Creates a copy of user with id: 3
+    //User(name=Alex, id=3)
+    //sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="tour-data-classes-copy-instance-kotlin"}
+
+### Access data class properties
+
+```kotlin
+data class User(val name: String, val id: Int)
+
+fun main() {
+    //sampleStart
+    val user = User("Alex", 1)
+    
+    println(user.component1()) //Prints first property of user
```

Comment:
I'm not sure, we really need `component1` and others here in examples. I think this section can be omitted. It's always better to access properties by their names. These `component*` functions are normally used for technical things (such as destructuring), not for regular code.